.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm run build
+      - run: |
+          git add --all && \
+          git diff-index --cached HEAD --stat --exit-code || \
+          (echo && echo "The above files changed because the build is not up to date." && echo "Please rebuild Prism." && exit 1)
+      - run: npm test
```

Comment:
We might as well do this now since GitAction workflows aren't easy to change (there are some restrictions where GH prevents you from pushing branches that change workflows).

Could you make `npm run build` + the git check a separate job similar to what [I did here](https://github.com/RunDevelopment/eslint-plugin-clean-regex/blob/886e4f25e0be5fbfacd42b3eb4b711d88a4bc6f8/.github/workflows/nodejs.yml#L45)? (Maybe call it `build`?)
The build check isn't really a test anyway.